test/integration/adapters/mailgun_test.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Swoosh.Integration.MailgunAdapterTest do
+  use ExUnit.Case, async: true
+
+  alias Swoosh.Adapters.Mailgun
+  import Swoosh.Email
+
+  @tag external: true
```

Comment:
Not seen `@moduletags` but judging by the ExUnit documentation `@tags` is for the module...